### PR TITLE
Update cartridge to 2.7.4 and metrics to 0.13.0

### PR DESCRIPTION
**Pull request by yngvar-antonsson**
I didn't forget about

- [x] Tests
- [x] Changelog

### CODE REVIEW

**Review by DifferentialOrange — APPROVED** [review]

**Comment by DifferentialOrange** [comment]
LGTM after green CI and rebased CHANGELOG

**Comment by yngvar-antonsson** [comment]
Blocked by #701

**Review by LeonidVas — APPROVED** [review]

**Pull request merged**
